Hrm, interesting bug failing the 1.31.0  build:

```
error[E0658]: imports can only refer to extern crate names passed with `--extern` on stable channel (see issue #53130)
  --> /home/mv/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.40/src/lib.rs:90:9
   |
89 | mod print;
   | ---------- not an extern crate passed with `--extern`
90 | pub use print::{BacktraceFmt, BacktraceFrameFmt, PrintFmt};
   |         ^^^^^
   |
note: this import refers to the module defined here
  --> /home/mv/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.40/src/lib.rs:89:1
   |
89 | mod print;
   | ^^^^^^^^^^
error[E0658]: imports can only refer to extern crate names passed with `--extern` on stable channel (see issue #53130)
  --> /home/mv/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.40/src/lib.rs:90:9
   |
89 | mod print;
   | ---------- not an extern crate passed with `--extern`
90 | pub use print::{BacktraceFmt, BacktraceFrameFmt, PrintFmt};
   |         ^^^^^
   |
note: this import refers to the module defined here
  --> /home/mv/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.40/src/lib.rs:89:1
   |
89 | mod print;
   | ^^^^^^^^^^
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,6 +1,6 @@
 language: rust
 rust:
-  - 1.31.0  # TODO: Adjust minimum supported Rust version accordingly
+  - 1.32.0  # TODO: Adjust minimum supported Rust version accordingly
   - stable
   - nightly
 cache: cargo

